app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
```

Comment:
- [ ] Please ensure that the form for login a user matches the one provided in the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_login.png), as stated in the requirement.

#### Requirement :point_down: 
![Screenshot from 2023-05-19 19-28-14](https://github.com/Chaseknis/blog_app/assets/89228743/a44ac0c5-363e-4b9c-98e9-046b4d813bdb)

### Your form so far 
![Screenshot from 2023-05-19 19-18-25](https://github.com/Chaseknis/blog_app/assets/89228743/0175fbe5-20a7-4572-a666-32d8385dec2f)